students/synowiec_krzysztof/lesson_10_organizing_files/filling_in_the_gaps.py

```diff
@@ -0,0 +1,36 @@
+import os
+import re
+import shutil
+
+
+BASE_DIR = "original"
+PREFIX = "spam"
+
+
+def main():
+    files = [file for file in os.listdir(BASE_DIR) if file.startswith(PREFIX)]
+    print(files)
+    files.sort()
+    print(files)
+    pattern = re.compile(r'\d{3}')
+
+    if len(files) > 2:
+        prev_n = None
+        gap_found = False
+        for file in files:
+            curr_n = int(pattern.search(file).group())
+            print("current {}".format(curr_n))
+            if not gap_found:
+                if prev_n is not None and prev_n + 1 != curr_n:
```

Comment:
I think it would be better to combine `if` from line 23 and 24 into single multiline `if`